src/nvim/ui_client.c

```diff
@@ -150,10 +150,9 @@ void ui_client_event_grid_line(Array args)
   Integer startcol = args.items[2].data.integer;
   Array cells = args.items[3].data.array;
 
-  Integer endcol, clearcol;
+  Integer clearcol;
```

Comment:
Maybe this declaration can also be moved down to there `clearcol` is first assigned.